pom.xml

```diff
@@ -166,6 +167,15 @@
                     <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Why are you defining the compiler plugin &  the property compiler specs. The `<maven.compiler.source>16</maven.compiler.source>` takes care of this interaction already.

Reply:
I added that as a temporary check to see if it compiles properly on java 16. After bumping the lombok verison, it compiles fine. This has been removed.